build_tools/rocksdb-lego-determinator

```diff
@@ -84,22 +84,21 @@ UPLOAD_DB_DIR="
     ]
 }"
 
-# We will eventually set the RATIO to 1, but we want do this
-# in steps. RATIO=$(nproc) will make it work as J=1
+# set default RATIO to 1, which sets J=$(nproc) and j=$(nproc)
 if [ -z $RATIO ]; then
-  RATIO=$(nproc)
+  RATIO=1
 fi
 
-if [ -z $PARALLEL_J ]; then
-  PARALLEL_J="J=$(expr $(nproc) / ${RATIO})"
+if [ -z $PARALLEL_VARS ]; then
+  # These cannot go after 'make' because of recursive make calls. Set them
```

Comment:
Actually this was a mis-diagnosis so I can go back to more like the old PARALLELISM setup